### PR TITLE
Remove user project header from local-run

### PR DESCRIPTION
**Pull request by cjellick**

### CODE REVIEW

**Comment by wlan0** [comment]
:+1: LGTM assuming build passes

**Comment by will-chan** [comment]
LGTM. I had this added on Friday already but did not submit the PR.

File: tools/create-service-account.sh [hunk 1]
```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
+# This script assumes auth is off
 # Create the account
-ID=$(curl  -X POST -H 'Accept: application/json' -H 'Content-Type: application/json' -H 'X-API-Project-Id: USER' -d '{"kind":"service", "name":"service", "uuid":"service"}' 'http://localhost:8080/v1/accounts' | jq -r '.id')
+ID=$(curl  -X POST -H 'Accept: application/json' -H 'Content-Type: application/json' -d '{"kind":"service", "name":"service", "uuid":"service"}' 'http://localhost:8080/v1/accounts' | jq -r '.id')
 
 # Create api keys for the account
 curl  -X POST -H 'Accept: application/json' -H 'Content-Type: application/json' -H "X-API-Project-Id: $ID" -d "{\"accountId\":\"$ID\", \"name\":\"service\", \"publicValue\":\"service\", \"secretValue\":\"servicepass\"}" 'http://localhost:8080/v1/apikeys'
```
Comment (wlan0) [inline tools/create-service-account.sh:8]: @cjellick  The X-API-Project-Id: $ID needs to be removed from the create apiKeys curl request as well.

**Comment by cjellick** [comment]
@wlan0 did it in his PR. Closing this one.

**Pull request closed without merging**